### DISCUSSION TITLE
Licence question

### DISCUSSION THREAD

**Discussion by aurelin** [nathanwoulfe/Plumber-2#101] [General]
Hi Nathan,

I've been looking into plumber/umbraco for our website redesign, and we are fairly convinced that this will work for us.
However, we are having a small problem, which is that our test environment is accessible through subdomains of one top level domain (say, domainA.tld), but our production actually lives under a completely different top level domain (say, domainB.tld).
The website under domainA.tld is the main site of the organisation we are part of, but is run completely separately by a different subdivision (and is powered by different software).

Is there a way we can use the production domain licence on these test/preprod subdomains?

Thanks!

**Comment by nathanwoulfe** [comment]
Hi @aurelin, the license is bound to a hostname, being the one used to access the backoffice. If your test environment is a subdomain of the licensed domain then the one license will work for both.

If the test env is served from a different domain, the license cannot be shared.

**Comment by aurelin** [comment]
Hi Nathan,

Thanks for the reply. I just need a couple more clarifications.
You say the licence is bound to the hostname of a system - our systems are named server.code.organisation.tld, so that's what the hostname is like. But the production site is accessed through department.organisation.tld. Is that going to work?

And, does the license work for sub-domains of any level (such that I could set up a like testing.sub.domain.tld)?

Thanks!

**Reply by nathanwoulfe** [reply]
Yup, that would all work fine, assuming the license was for organisation.tld - any subdomain will be covered. 

If you have multiple Umbraco instances running on different subdomains of the same host, one license will cover all instances.
